docs/UserGuide.md

```diff
@@ -289,14 +289,19 @@ Format:
 
 Examples:
 * `add property n/Blk 123 a/123, Clementi Rd, #04-20, 1234665 s/James Lee p/61234567 e/james@email.com $/100000 t/hdb t/3rm`
-* `add property n/Blk 321 a/123, Clementi Rd, #04-20, 1234665 s/James Lee p/(hp) 61234567 e/james@email.com $/100000 t/hdb t/3rm`
 * `add buyer n/Sam p/91234567 e/sam@email.com $/740000 t/hdb t/3rm`
 
+Shown below is the output for the valid input `add property n/Blk 321 a/123, Clementi Rd, #04-20, 1234665 s/James Lee p/(hp) 61234567 e/james@email.com $/100000 t/hdb t/3rm`.
+
+![Add Property](images/AddValidUi.png)
+
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A property/buyer can have any number of tags (including 0). All tags will be converted to lowercase.
+If you typed in an invalid input, the command box will demonstrate the correct input format so you can correct your input,
+such as in the case below.
+<br />
+<img src="images/AddInvalidUi.png" width="600"/>
```

Comment:
Should crop this picture to just the command box and output.
Not directly relevant to this PR: We should change add buyer message usage to $/BUDGET, has this been done somewhere else?